### DISCUSSION TITLE
Confusion around PCA

### DISCUSSION THREAD

**Discussion by rushild8** [ChristianHuber/smartsnp#3] [Q&A] [answered]
Hello,

I am trying to use Christian Huber's smartsnp package to form a PCA projection. I was following the tutorial until the actual smartpca section, entitled "Running smartpca". I am very confused as to what is going on in this section. Did we not already remove unwanted data through the par.ANCESTRYMAP.FILTER step? What are these vectors that are being made and how are they relevant? I would really appreciate if someone could provide a breakdown on what is happening in this step.

Thank you,
Rushil

**Comment by ChristianHuber** [comment, marked as answer]
Hi Rushil,
The par.ANCESTRYMAP.FILTER step is only filtering the modern data, it is sub-selecting West Eurasian populations. However, the ancient data also contains some individuals that we don't want in the PCA, e.g. the chimp sequence, the hg19 reference, archaic human sequences etc. The smart_pca function allows you to filter out individuals by providing the row number of these individuals (you get the row numbers with the "which" statement). You could probably also use convertf to filter the ancient data first before using smart_snp.
Hope this helps!
Christian
